main/client.go

```diff
@@ -43,6 +44,8 @@ type KeyRegistration struct {
 	ValidNotBefore string `json:"validNotBefore"`
 }
 
+// SignedKeyRegistration is a signed certificate registered with the
+// key-service.
 type SignedKeyRegistration struct {
```

Comment:
I don't know if i understood this correctly; KeyRegistration is when a public key is registered with the key-service, and SignedKeyRegistration is the same, but signed (by whom?)?